tox.ini

```diff
@@ -110,3 +110,16 @@ deps =
     -r{toxinidir}/requirements.txt
 commands =
     pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}
+
+[testenv:integration-juju-2-9]
```

Comment:
I think tox supports parametrizing the version of a package, see: https://github.com/jdkandersson/flake8-docstrings-complete/blob/d63a6d833912f773ed2517fa4cf675686b7b91e8/tox.ini#L53

Reply:
Done